internal/guestrequest/types.go

```diff
@@ -119,3 +120,8 @@ const (
 type SignalProcessOptionsWCOW struct {
 	Signal SignalValueWCOW `json:",omitempty"`
 }
+
+type HvSocketAddress struct {
+	LocalAddress  string `json:",omitempty"`
+	ParentAddress string `json:",omitempty"`
+}
```

Comment:
Is this from a mars file? Should we be generating this instead off hand writing it?

Reply:
It is from just checking. This is the same signature but maybe put this in /internal/schema2 with the other hvsocket generated files.

Reply:
If we move this to a file that is generated without it typically, please add a comment saying this struct is not generated by default. We have an open item to improve our schema generation in the future :)

Reply:
Me or Amit can generate it and put it with the rest, but yes that should be the goal :P

Reply:
I have added a newly generated file in the schema2 folder. (Thanks to @dcantah for generating the file).